.gitignore

```diff
@@ -30,6 +30,9 @@ Pods
 # Carthage
 Carthage/
 
+# Swift Package Manager
+.build/
+
```

Comment:
Since this is going to depend on SwiftCheck its going to have a Packages directory. Should that be ignored as well?